pom.xml

```diff
@@ -0,0 +1,129 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.systelab</groupId>
+    <artifactId>java-report-test-utilities</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+    <name>java-report-test-utilities</name>
+    <url>https://github.com/systelab/java-report-test-utilities</url>
+    <description>Report Test Utilities Library</description>
+    <scm>
+        <url>https://svn.apache.org/viewvc/maven</url>
+    </scm>
+    <prerequisites>
+        <maven>2.2.0</maven>
+    </prerequisites>
+    <profiles>
+        <profile>
+            <id>github</id>
+            <distributionManagement>
+                <repository>
+                    <id>github</id>
+                    <name>GitHub Packages</name>
+                    <url>https://maven.pkg.github.com/systelab/java-report-test-utilities</url>
+                </repository>
+            </distributionManagement>
+        </profile>
+    </profiles>
+    <licenses>
+        <license>
+            <name>Unlicense</name>
+            <url>http://unlicense.org</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <jasperrport.version>6.17.0</jasperrport.version>
```

Comment:
Why do we need to import jasperreport here? If I'm not wrong, to compare 2 PDF files, we just use pdfbox at this point. If so, I would suggest cleaning up this dependency.

Reply:
Good catch, I forgot to remove it.